### DISCUSSION TITLE
Nötige Ausgaben

### DISCUSSION THREAD

**Discussion by sp33dlink** [Hartz-Arbeit/LohntDat#28] [Allgemein]
Da wir hier ja ein Open Source Freizeitprojekt sind:
Wie stemmen wir die Kosten für notwendige Ausgaben?
Mir fallen da direkt ein:
* Webhosting (5€ monatl. ?)
* Entwicklerlizenzen App-Stores (Google Play 25€ einmalig, Apple Store 99€ jährl.)

Später bestimmt machbar per Spendenbutton aber wer buttert da initial was zu?

Btw: als gemeinnützige Organisationen kann man sich die 99€ Apple Gebühren im Jahr sparen

**Comment by tundrablasen** [comment]
Kann das nicht über eine bestehende Lizenz online gehen? War nicht initial einen Web App geplant? Eine store App ist glaube ich erstmal zu groß. Eine gut programmierte Web App die mobile First ausgerichtet ist, sollte doch erstmal reichen. 

Besonders weil die Hürde einer App Installation wieder viel größer ist als einen link aufzurufen

**Comment by thraizz** [comment]
Hosting bekommen wir mit Firebase kostenlos hin, ich sehe unausweichliche Kosten nur bei der saucoolen Domain.

**Comment by pkunze** [comment]
Wenn wir es als (static) WebApp deployen haben wir keine Kosten, dann hosten wir es als GitHub Pages (wie auch im Unter angeregt). Wäre jedenfalls die einfachste und günstigste Lösung (und ich meine bspw. SchnellTestTest macht es ebenso, funktioniert und wird angenommen).

**Comment by tundrablasen** [comment]
Projekte hab ich bisher immer mit [Bitpalast](https://cheap-webserver-us.bitpalast.net/)  realisiert. Domain inkl und Webspace sowieso. Für 1,80€ im Monat

**Comment by frieswiththat** [comment]
> GitHub Pages

+1 dafür, insbesondere in Verbindung mit dem bestehenden Repo. Je weniger Abhängigkeiten nach externen Services desto schneller geht das ganze live. Domain kann immer noch drüber gebügelt werden.

**Comment by Hartz-Arbeit** [comment]
Eine Tatsächliche App im Appstore ist auch erstmal übertrieben würde ich sagen. Das ist für manche vielleicht auch eine Hemmschwelle nur wegen nem Calculator eine App zu installieren.

**Comment by Aplikator90** [comment]
Ich könnte bei paar Stiftungen nach einer Finanzierung nachfragen. Bis 2000€ funktioniert es relativ einfach. Bürgerstiftung Stuttgart oder Vector Stiftung sind da immer sehr offen. Aber dann hat man die an der Backe, für die Kosten die genannt wurden wäre es nicht notwendig. 

Bei der Frage app oder nicht app stellt sich für mich die Frage der Zielgruppe. Erfahrungsgemäß besitzen fast alle Jobcenter Kunden ein Smartphone aber eher selten ein Computer. Daher wäre die Ansicht aus dem mobilen Browser wichtiger als die im normalen. Falls die Zielgruppe aber Menschen sind die es aus Interesse durchrechnen möchten ist das was anderes.

**Comment by jpv-os** [comment]
> > GitHub Pages
> 
> +1 dafür, insbesondere in Verbindung mit dem bestehenden Repo. Je weniger Abhängigkeiten nach externen Services desto schneller geht das ganze live. Domain kann immer noch drüber gebügelt werden.

Hier pflichte ich auch stark bei, die Hostingkosten können durch Github Pages komplett eliminiert werden und es ist insbesondere keine Privatperson nötig, um das Vertragliche zu regeln und im Zweifelsfall auch den Kopf hinzuhalten. Über Github Pages können die Maintainer des Repositories das zunächst "offen" handhaben. 

Und wenn dann doch eine Domain gekauft wird, dann kann diese von einer Privatperson/Firma/Verein/Organisation/... erworben werden und auf die Github Pages Seite zeigen.

**Comment by cr4yfish** [comment]
>Daher wäre die Ansicht aus dem mobilen Browser wichtiger als die im normalen

100%. 

>Und wenn dann doch eine Domain gekauft wird, dann kann diese von einer Privatperson/Firma/Verein/Organisation/... erworben werden und auf die Github Pages Seite zeigen.

Also die "_lohntdat.de_" Domain kann ich heute Abend kaufen - kostet ja schließlich <1€/Monat. Wir müssen die ja nicht sofort mit einbinden für den MVP, aber reserviert ist sie dann schonmal.
